scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Example: Elasticsearch Input and Output
```

Comment:
Can you expand this doc a little bit on what the example does?

Reply:
Also the name / title is a bit misleading. We don't actually support ES input.